handleData.js

```diff
@@ -119,11 +169,43 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = {};
+    this.money = { "Copper"      : 0,
```

Comment:
I think this might be cleaner to read if you aligned the keys on the next line.

```javascript
this.money = {
  "Copper" : 0,
  "Silver": 0,
  ...
};
```